### PR TITLE
Main: Add Wiby Search Engine

### PR DESCRIPTION
**Pull request by kimkulak**
Wiby is a search engine for older style pages, lightweight and based on a subject of interest. Building a web more reminiscent of the early internet.

### CODE REVIEW

**Comment by circl-lastname** [comment]
`Main:` should be `Base:` and the merge commit shouldn't be there, rebase onto the branch instead

**Comment by kimkulak** [comment]
> `Main:` should be `Base:` and the merge commit shouldn't be there, rebase onto the branch instead

Fixed :^)

File: Base/home/anon/.config/SearchEngines.json [hunk 1]
```diff
@@ -27,6 +27,10 @@
         "title": "Mojeek",
         "url_format": "https://www.mojeek.com/search?q={}"
     },
+    {
+        "title": "Wiby",
+        "url_format": "http://wiby.me/?q={}"
```
Comment (linusg) [inline Base/home/anon/.config/SearchEngines.json:32]:
```suggestion
        "url_format": "https://wiby.me/?q={}"
```

**Review by linusg — CHANGES_REQUESTED** [review]

**Review by linusg — COMMENTED** [review]
> Fixed :^)

Still says `Main:`.

**Comment by kimkulak** [comment]
> > Fixed :^)
> 
> Still says `Main:`.

I fixed it a couple of hour ago but managed to somehow both reverse the change and requested a review from you.
At least after this snafu things should work fine... right?

:^)

File: Base/home/anon/.config/SearchEngines.json (hunk 1 above)
Comment (linusg) [reply Base/home/anon/.config/SearchEngines.json:32]: This is not resolved

**Comment by linusg** [comment]
Please squash the fixup commit as per https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md#code-submission-policy

**Comment by kimkulak** [comment]
> Please squash the fixup commit as per https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md#code-submission-policy

This sounds easy enough :^)

**Comment by linusg** [comment]
Now it's saying `Update:` and some nonsensical commit message - please sort this out locally and carefully check before force pushing and requesting a review again.

**Comment by circl-lastname** [comment]
Guess they deleted the entire account and will try again? 🤔

**Comment by linusg** [comment]
Welp, that escalated quickly. They did try again, and I merged it: https://github.com/SerenityOS/serenity/pull/19737

**Pull request closed without merging**